atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
I think the confusion is that `koji_import` needs to know the list of all content_types available. And `pulp_pull`, at least for now, only needs to know whether or not v2 schema 2 is available.

Making `pulp_pull` perform the checks seems reasonable to me. We should that first, if that's the desired approach. 

To query the registry, there's the helper method: `util.get_manifest_digests`
But that only queries for v2 (both schema 1 and schema 2). 
Something similar must be added for v1 content, as I'm not aware of existing code that does this.